README.md

```diff
@@ -275,6 +276,56 @@ If you have used the `--redux` generator option, you will notice the familiar ad
 
 Note the organizational paradigm of "bundles". These are like application domains and are used for grouping your code into webpack bundles, in case you decide to create different bundles for deployment. This is also useful for separating out logical parts of your application. The concept is that each bundle will have it's own Redux store. If you have code that you want to reuse across bundles, including components and reducers, place them under `/client/app/lib`.
 
+#### Multiple React Components on a Page with One Store
+You may wish to have 2 React components share the same the Redux store. For example, if your navbar is a React component, you may want it to use the same store as your component in the main area of the page. You may even want multiple React components in the main area, which allows for greater modularity. In addition, you may want this to work with Turbolinks to minimize reloading the JavaScript.
+
+Suppose the Redux store is called `appStore`, and you have 2 React top level components, `Comments` and `Blogs`.
+
+**app/views/layouts/application.html.erb**
+```erb
+...
+<% initialize_store("appStore", props) %>;
+<%= render_component("Header") %>
+yield
+...
+```
+
+**_comments.html.erb**
+```erb
+<%= render_component("Comments") %>
+```
+
+**_blogs.html.erb**
+```erb
+<%= render_component("Blogs") %>
+```
+
+You will first need to make function that can create the store you will be using for all components and register it via the `registerStoreCreator` method. Note, this is a **storeCreator**, meaning that it is a function that takes props and returns a store:
+
+```
+ReactOnRails.registerStoreCreator({
+  appStoreCreator
+});
+```
+
+When registering your component with React on Rails, you can get the store via `ReactOnRails.getStore`:
+
+```js
+// getStore will initialize the store if not already initialized, so creates or retrieves store
+const appStore = ReactOnRails.getStore("appStore");
+return (
+  <Provider store={appStore}>
+    <NonRouterCommentsContainer />
+  </Provider>
+);
+```
+
+From your Rails view, you can use the provided helper `initialize_store(store_name, props)` to create a fresh version of the store (because it may already exist if you came from visiting a previous page).
+
+Components are created as [stateless function(al) components](https://facebook.github.io/react/docs/reusable-components.html#stateless-functions). Since you can pass in initial props via `initializeStore`, you do not need to pass any props directly to the component. Instead, the component hydrates by connecting to the store.
```

Comment:
Don't confuse `stateless function component` with `pure render component`. They are slightly different. Technically, I can still use the ES6 class syntax to create a component, use a pure render mixin, and it's a pure render component. A stateless function component specifically means a function that just returns some JSX. I linked to the corresponding section in the React docs.